src/hooks/usePoolData.ts

```diff
@@ -247,6 +254,20 @@ export default function usePoolData(name?: string): PoolDataHookReturnType {
           expandedPool.poolAddress
         ] || { oneDayVolume: null, apy: null, utilization: null }
 
+        let [amountStakedInMinichefUSD, minichefSDLApr] = [Zero, Zero]
```

Comment:
Since each of these are only assigned once, I don't think we need to create a tuple. We can probably extract this logic into a helper function that takes in `(priceDataForPool, poolMinichefData, tokenPricesUSD.SDL)` and just return the `minichefSDLApr`. We can set `const minichefSDLApr = getMinichefSDLApr(priceDataForPool, poolMinichefData, tokenPricesUSD.SDL) || Zero` if we want a default value